### PR TITLE
Add incidence as a primary variable

### PR DESCRIPTION
**Pull request by kairstenfay**
Previously, what we're calling "Incidence" (but is really
just count of participants) was missing from the viz
prototype. Add a temporary solution to make 
"Incidence" work with the existing viz.

I describe this commit as "temporary" because all
these if/else and switch/case statements are red flags
to me, signaling that the architecture of this app
should be rewritten, perhaps in an OO-design.

I welcome thoughts and feedback on current & future design here.

### CODE REVIEW

**Comment by jameshadfield** [comment]
Thanks @kairstenfay 🎉 i'll take a look shortly

While it's in your memory, could you elaborate a bit more on what you had in mind for a more "OO-design"?

**Comment by kairstenfay** [comment]
I'm thinking an object oriented (OO) design using polymorphism or
dependency injection. I don't have a lot of experience in OO programming,
though, so we should also consult @tsibley.

File: src/components/geo/render.js [hunk 1]
```diff
@@ -71,25 +71,29 @@ export const renderMap = ({ref, width, height, resultsData, modelViewSelected, s
       resultsData.counts.forEach((d) => {demeValues[d.key] = d[vizCategory];});
       domainMax = resultsData.maxValue;
     } else {
-      resultsData.percentages.forEach((d) => {demeValues[d.key] = d[vizCategory];});
+      resultsData.mapData.forEach((d) => {demeValues[d.key] = d[vizCategory];});
       domainMax = 100;
     }
 
-    if (!modelViewSelected) {
+    if (categories.length === 1) {
```
Comment (tsibley) [inline src/components/geo/render.js:78]: This feels fragile to me. What if another categorical variable happens to only have one level?

**Review by tsibley — APPROVED** [review]
I know we've de-prioritized this, but I saw it on Trello and wanted to make a decision on it. This seems to work for me and so I think fine to merge. I do have one comment about robustness, but ok with me to merge and move on for now.

File: src/components/geo/render.js (hunk 1 above)
Comment (kairstenfay) [reply src/components/geo/render.js:78]: I agree. I think the data-flow here needs an overhaul in general.

**Pull request merged**